front/public/locales/en/simulation.json

```diff
@@ -37,14 +37,19 @@
   "resetTimer": "Reset timer",
   "rotate": "Rotate the chart",
   "simplify": "Data simplification",
+  "speedSpaceChart": "Speed Space Chart",
   "speedSpaceSettings": {
-    "display": "Display",
-    "altitude": "Altitude",
-    "curves": "Curves",
-    "maxSpeed": "Line speed",
-    "slopes": "Slopes",
-    "electricalProfiles": "Electric traction",
-    "powerRestriction": "Power restriction"
+    "reticleInfos": "Reticle infos",
```

Comment:
Could you order the settings key alphabetically ?

Reply:
you have the vscode addon **Sort JSON objects**

Reply:
done